src/scripts/main.js

```diff
@@ -357,4 +357,26 @@ const people = [
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const table = document.querySelector('table');
+const th = table.tHead.rows[0].cells[1];
+const targetTh = table.tHead.rows[0].cells[4];
+const newTh = table.tHead.rows[0].insertCell(5);
+
+newTh.outerHTML = targetTh.outerHTML;
+targetTh.innerHTML = th.innerHTML;
+
+const tf = table.tFoot.rows[0].cells[1];
+const targetTf = table.tFoot.rows[0].cells[4];
+const newTf = table.tFoot.rows[0].insertCell(5);
+
+newTf.outerHTML = targetTf.outerHTML;
+targetTf.innerHTML = tf.innerHTML;
+
+for (let row = 0; row < table.tBodies[0].rows.length; row++) {
+  const cell = table.tBodies[0].rows[row].cells[1];
+  const targetCell = table.tBodies[0].rows[row].cells[4];
+  const newCell = table.tBodies[0].rows[row].insertCell(5);
```

Comment:
The loop assumes that each row in the table body has at least 5 columns. Verify that this is the case to avoid runtime errors.